sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -79,8 +79,10 @@
 PYTORCHJOB_PLURAL = "pytorchjobs"
 PYTORCHJOB_CONTAINER = "pytorch"
 PYTORCHJOB_REPLICA_TYPES = (REPLICA_TYPE_MASTER.lower(), REPLICA_TYPE_WORKER.lower())
-
 PYTORCHJOB_BASE_IMAGE = "docker.io/pytorch/pytorch:1.12.1-cuda11.3-cudnn8-runtime"
+PYTORCHJOB_STORAGE_CONTAINER = "pytorch-storage"
+PYTORCHJOB_STORAGE_CONTAINER_IMAGE = "docker image path"
```

Comment:
Do we need to separate storage initializer per framework ?
I guess, we are going to use single storage init image for all frameworks (e.g. PyTorch, MPI).